test/helpers/assertThrow.js

```diff
@@ -0,0 +1,71 @@
+const { assert } = require('chai')
```

Comment:
We have to include a custom `assertThrow` file to handle a new error message format that hardhat returns and `@aragon/contract-helpers-test` do not handle it.